src/core/entities/MRDivEntity.js

```diff
@@ -105,11 +105,11 @@ export class MRDivEntity extends MREntity {
      * @description Removing an entity as a sub-object of this panel (for example an mr-model, button, etc).
      * @param {MREntity} entity - the entity to be removed added.
      */
-    remove(entity) {
+    removeEntity(entity) {
         // `this` must have `mr-panel` as its closest parent entity for threejs to handle positioning appropriately.
         let panel = this.closest('mr-panel');
         if (panel && entity instanceof MRDivEntity) {
-            panel.remove(entity);
+            panel.removeEntity(entity);
         } else {
             this.object3D.remove(entity.object3D);
```

Comment:
this grab is for when an MRDivEntity is a child of an MRPanelEntity. In order for layout to work we had to flatten the hierarchy within an MRPanelEntity, but there's a chance that the user will create an MRDivEntity outside of an MRPanelEntity, so this handles that case.

Reply:
Discussed on discord:

- concern was that allowing `mr-div` to exist outside an `mr-panel` could lead to confusion. but _not_ allowing it could limit creativity
- current solution: We only document `mr-div` usage within `mr-panel` and observe developer feedback as the library matures.